docs/deploy_streamlit_app.md

```diff
@@ -56,7 +56,7 @@ for example:
 http://share.streamlit.io/streamlit/demo-self-driving/master/streamlit_app.py
 ```
 
-Most times, your app is also given a shortened URL of the form `https://share.streamlit.io/[user name]/[repo name]`. The only time you need the full URL is when you deployed multiple apps from the same repo. So you can also reach the example URL above at the short URL [http://share.streamlit.io/streamlit/demo-self-driving](http://share.streamlit.io/streamlit/demo-self-driving).
+Most times, if your app has name `streamlit_app.py` and your branch is `master`, your app is also given a shortened URL of the form `https://share.streamlit.io/[user name]/[repo name]`. The only time you need the full URL is when you deployed multiple apps from the same repo. So you can also reach the example URL above at the short URL [http://share.streamlit.io/streamlit/demo-self-driving](http://share.streamlit.io/streamlit/demo-self-driving).
```

Comment:
Since this is so specific to the times when it occurs, should the beginning of the sentence remove "Most times"? If not, what are the situations when this doesn't occur automatically?

Reply:
I remembered why I kept it in... the sentence following shows another exception. But since that sentence remains, I agree it makes more sense to remove!